packages/cardano-services/src/Asset/CardanoTokenRegistry.ts

```diff
@@ -112,14 +112,14 @@ export class CardanoTokenRegistry implements TokenMetadataService {
   }
 
   async getTokenMetadata(assetIds: Cardano.AssetId[]) {
-    this.#logger.debug(`Requested asset metatada for "${assetIds}"`);
+    this.#logger.debug(`Requested asset metadata for "${assetIds}"`);
```

Comment:
These spelling mistake corrections should be committed separately as a `fix` commit.

Reply:
Split into a separate commit: https://github.com/input-output-hk/cardano-js-sdk/pull/490/commits/ae19731896fdcef2f95627b382ddb63b5a7925dc